src/config/validators.ts

```diff
@@ -5,6 +5,11 @@ type ValidatorInfo = {
 };
 
 export const VALIDATOR_GROUPS: Record<string, ValidatorInfo> = {
+  '0x89d5bd54C43dDd10905A030DE6Ff02EbB6c51654': {
+    logo: '/logos/validators/polychain.jpg',
+    url: 'https://polychain.capital',
+    twitter: 'https://twitter.com/polychain',
+  },
```

Comment:
There's already a polychain entry but the incorrect address was used. Can you change the `0x47b2dB6af05a55d42Ed0F3731735F9479ABF0673` address below to `0x89d5bd54C43dDd10905A030DE6Ff02EbB6c51654`? Thanks for the contribution!

Reply:
Sure thing, updated!